src/runtime/virtcontainers/sandbox.go

```diff
@@ -1206,14 +1242,12 @@ func (s *Sandbox) startVM(ctx context.Context, prestartHookFunc func(context.Con
 
 	s.Logger().Info("Starting VM")
 
-	if s.config.HypervisorConfig.Debug {
```

Comment:
Seems we could not remove this line. The serial device (console channel) are created based on `HypervisorConfig.Debug`, which means, if this param is not set, we can not reading kata-agent's log (?

Reply:
We need the consoleWatcher to read the kata-agent's log, you are correct. My PR however makes the consoleWatcher always enabled. As I made in the PR description, if the user thinks there are too many logs from kata-agent, the user can use the kata-agent's log level to tune.